datatypes.go

```diff
@@ -103,6 +104,18 @@ func (event *NotificationEvent) CreateMessage(location *time.Location) string {
 // NotificationEvents represents slice of NotificationEvent
 type NotificationEvents []NotificationEvent
 
+type TemplateEvent struct {
```

Comment:
The name of this struct do not explain what this struct is about. Let's rename it to `EventTemplateData`